src/app/view/mouseAndKeyboardHandler.js

```diff
@@ -106,13 +106,13 @@ export default class MouseAndKeyboardHandler {
             }
             // Copy
             if ((event.ctrlKey || event.metaKey) && key == 67) {
-                //console.log("Ctl c detected");
+                // console.log("Ctl c detected");
                 reference.initiateCopy();
             }
             // Cut
             if ((event.ctrlKey || event.metaKey) && key == 88) {
                 //console.log("Ctl x detected");
-                let selectedFeatures = reference.view.getSelectedFeatures();
+                let selection = reference.view.getSelectedFeatures();
                 if (selectedFeatures.length > 0) {
                     reference.pasteboard[0] = selectedFeatures[0];
                 }
```

Comment:
It seems like the cut system also got modified during the variable change but its not using the latest selection system

Reply:
Ok I can change that to also use the recent selection system